ml-testing-toolkit/values.yaml

```diff
@@ -16,6 +16,13 @@ ml-testing-toolkit-backend:
   #       port: 5050
   #       paths: ['/api/', '/socket.io/']
   config:
+    ## We can specify the files and folders here those we want to override in the TTK `spec_files` directory.
```

Comment:
Should we not also include this in the same section within the `mojaloop/values.yaml`?

Reply:
Done